composer.json

```
@@ -38,9 +38,9 @@
         "ext-json": "*",
         "illuminate/contracts": "^6.0|^8.0|^9.0",
         "illuminate/support": "^6.0|^8.0|^9.0",
-        "laragraph/utils": "^1",
+        "laragraph/utils": "1.4.1",
```

```suggestion
        "laragraph/utils": "dev-graphql-php-15",
```

Should be the easier temporary fix.